packages/network/lib/cors.ts

```diff
@@ -227,14 +227,11 @@ export function getOrigin (url: string) {
 }
 
 /**
- * Returns the super-domain of a URL
- *
- * The primary driver uses the super-domain origin to allow tests to
- * navigate between subdomains of the same super-domain by setting
- * document.domain to the super-domain
+ * We use the super domain origin in the driver to determine whether or not we need to reload/interact with the AUT, and
+ * currently in the spec bridge to interact with the AUT frame, which uses document.domain set to the super domain
```

Comment:
I think the comment update is still valid @chrisbreiding. I'd like to keep those changes and will update with a commit

Reply:
fixed in 070ba434bf